pydm/widgets/timeplot_qtplugin.py

```diff
@@ -1,5 +1,75 @@
-from .qtplugin_base import qtplugin_factory
+from ..PyQt.QtDesigner import QExtensionFactory, QPyDesignerTaskMenuExtension
+from ..PyQt.QtGui import QAction
+from ..PyQt.QtCore import pyqtSlot
+from .qtplugin_base import PyDMDesignerPlugin
 from .timeplot import PyDMTimePlot
+from timeplot_curve_editor import TimePlotCurveEditorDialog
```

Comment:
I only got this line working by relative import (adding the dot):

`from .timeplot_curve_editor import TimePlotCurveEditorDialog`